### PR TITLE
Add Django 2.0 support

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+exclude = migrations,docs,env,.tox
+max-line-length = 120
+[bdist_wheel]
```
Comment (ironfroggy) [inline setup.cfg]: Can we add a blank line between sections

File: stickyuploads/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 "Enhanced file input widget for Django which uploads the file in the background and retains value on form errors."
 
-__version__ = '0.6.1'
+__version__ = '0.7.0'
```
Comment (ironfroggy) [inline stickyuploads/__init__.py]: Sticky uploads has been around for a long time and this version breaks some major backwards compatibility. As good a time as any to cut a 1.0

File: tox.ini [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [tox]
 # Assumption: if it works on py34 and py36, py35 will be okay too.
 # But when dropping py34, put py35 back into the test matrix.
-envlist = {py27,py34,py36}-{1.11,1.10,1.8}.X,docs
+envlist = {py34,py36}-2.0.X,{py27,py34,py36}-1.11.X,docs,flake8
```
Comment (ironfroggy) [inline tox.ini]: Can we test 3.5 too, rather than skipping it between 3.4 and 3.6

**Review by ironfroggy — CHANGES_REQUESTED** [review]

**Comment by dpoirier** [comment]
Good suggestions @ironfroggy . Updated.

**Review by ironfroggy — APPROVED** [review]

**Pull request merged**